Adds CLI for Microsoft 365 v11.1 release blog post (#2197)
pnp/blog: content/cli-for-microsoft-365/cli-for-microsoft-365-v11-1/index.md

## New commands

### SharePoint list alerts

In this release, we added two new commands that allow you to list all SharePoint list alerts and remove an existing alert. Let's go over a few examples of how to use these new commands:

In order to list all SharePoint list alerts in a specific site, run:

```sh
m365 spo web alert list --webUrl https://contoso.sharepoint.com/sites/Marketing
```

To list alerts for a SharePoint list by listUrl:

```sh
m365 spo web alert list --webUrl https://contoso.sharepoint.com/sites/Marketing --listUrl /sites/Marketing/lists/Tasks
```

To list alerts for a specific list by title and user by username, execute the following:

```sh
m365 spo web alert list --webUrl https://contoso.sharepoint.com/sites/Marketing --listTitle "Tasks" --userName jane.doe@contoso.com
```

> It is possible to remove an alert by its ID using the following command:

Author: ```suggestion
To remove an existing alert, execute the following:
```
